src/components/feedback/feedbackForm.tsx

```diff
@@ -53,12 +55,17 @@ export function FeedbackForm({onClose, onSubmit}: FeedbackFormProps) {
         />
       </Label>
       <Label htmlFor="sentry-feedback-comment">
-        Description
+        <div>
+          Description<Required>*</Required>
+        </div>
         <TextArea
           rows={5}
+          onChange={event => {
+            setHasDescription(!!event.target.value);
```

Comment:
is the `!!` intentional😮

Reply:
Yeah, it's a shorthand for casting the string to a boolean (rather than `Boolean()`)

Reply:
TIL